docs/api-inference/tasks/image-text-to-text.md

```diff
@@ -0,0 +1,129 @@
+- the "tasks" content for recommended models: https://huggingface.co/api/tasks
+--->
+
+## Image-Text to Text
+
+Image-text-to-text models take in an image and text prompt and output text. These models are also called vision-language models, or VLMs. The difference from image-to-text models is that these models take an additional text input, not restricting the model to certain use cases like image captioning, and may also be trained to accept a conversation as input.
+
+<Tip>
+
+For more details about the `image-text-to-text` task, check out its [dedicated page](https://huggingface.co/tasks/image-text-to-text)! You will find examples and related materials.
+
+</Tip>
+
+### Recommended models
+
+- [HuggingFaceM4/idefics2-8b-chatty](https://huggingface.co/HuggingFaceM4/idefics2-8b-chatty): Cutting-edge conversational vision language model that can take multiple image inputs.
+- [microsoft/Phi-3.5-vision-instruct](https://huggingface.co/microsoft/Phi-3.5-vision-instruct): Strong image-text-to-text model.
+
+This is only a subset of the supported models. Find the model that suits you best [here](https://huggingface.co/models?inference=warm&pipeline_tag=image-text-to-text&sort=trending).
+
+### Using the API
+
+
+<inferencesnippet>
+
+<curl>
+```bash
+curl https://api-inference.huggingface.co/models/HuggingFaceM4/idefics2-8b-chatty \
+	-X POST \
+	-d '{"inputs": No input example has been defined for this model task.}' \
+	-H 'Content-Type: application/json' \
+	-H "Authorization: Bearer hf_***"
+```
+</curl>
+
+<python>
+```py
+import requests
+
+API_URL = "https://api-inference.huggingface.co/models/HuggingFaceM4/idefics2-8b-chatty"
+headers = {"Authorization": "Bearer hf_***"}
+
+from huggingface_hub import InferenceClient
+
+client = InferenceClient(api_key="hf_***")
+
+image_url = "https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg"
+
+for message in client.chat_completion(
+	model="HuggingFaceM4/idefics2-8b-chatty",
+	messages=[
+		{
+			"role": "user",
+			"content": [
+				{"type": "image_url", "image_url": {"url": image_url}},
+				{"type": "text", "text": "Describe this image in one sentence."},
+			],
+		}
+	],
+	max_tokens=500,
+	stream=True,
+):
+	print(message.choices[0].delta.content, end="")
+```
+
+To use the Python client, see `huggingface_hub`'s [package reference](https://huggingface.co/docs/huggingface_hub/package_reference/inference_client#huggingface_hub.InferenceClient.image_text-to-text).
+</python>
+
+<js>
+```js
+async function query(data) {
+	const response = await fetch(
+		"https://api-inference.huggingface.co/models/HuggingFaceM4/idefics2-8b-chatty",
+		{
+			headers: {
+				Authorization: "Bearer hf_***"
+				"Content-Type": "application/json",
+			},
+			method: "POST",
+			body: JSON.stringify(data),
+		}
+	);
+	const result = await response.json();
+	return result;
+}
+
+query({"inputs": No input example has been defined for this model task.}).then((response) => {
```

Comment:
Same issue here with idefics 8b chatty not been detected as "conversational". But at least the JS snippet is not buggy :smile:

Reply:
`HuggingFaceM4/idefics2-8b-chatty` was not detected since it does not have the `conversational` tag (also something to fix).

Reply:
EDIT: I tried adding the tag manually, since it cannot be detected in moon-landing, but it seems I don't have write permission to create a branch in https://huggingface.co/HuggingFaceM4/idefics2-8b-chatty

Reply:
> EDIT: I tried adding the tag manually, since it cannot be detected in moon-landing, but it seems I don't have write permission to create a branch in https://huggingface.co/HuggingFaceM4/idefics2-8b-chatty

Did you try opening a PR in https://huggingface.co/HuggingFaceM4/idefics2-8b-chatty/discussions?new_pr=true ?

Reply:
I wanted to add the tag with a request to `/api/models/HuggingFaceM4/idefics2-8b-chatty/tag/` but I got permission error, so I opened a PR with https://huggingface.co/HuggingFaceM4/idefics2-8b-chatty/discussions?new_pr=true and added the tag in the README : [HuggingFaceM4/idefics2-8b-chatty/discussions/11](https://huggingface.co/HuggingFaceM4/idefics2-8b-chatty/discussions/11).

Reply:
@hanouticelina There is a confusion between the `/tag` API and tags in the model cards. The API one is used to create a tag in the git history of the repo. The tags in the model cards are what you did in the PR which is adding some metadata to the repo on the Hub (for search, discoverabiility, etc.).

So all good to have created a PR, that's the way of doing it!

Reply:
Ah okay! makes sense, thanks for clarifying this